pages/components/rating/rating.css

```diff
@@ -0,0 +1,16 @@
+@import "../../../styles/typography.css";
+@import "../../../styles/variable.css";
+@import "../../../styles/reset.css";
+@import "../../../styles/header.css";
+@import "../../../styles/drawer.css";
+@import "../../../styles/padding.css";
+@import "../../../styles/margin.css";
+@import "../../../styles/utils.css";
+
+.simple-rating-wrapper span {
+  cursor: pointer;
+}
+
+.checked {
+  color: orange;
+}
```

Comment:
maybe use colours in hexcode, i got this suggestion so sharing. If i am wrong ignore it :)

Reply:
Use intentionally, for others I have declared them in variables.